### PR TITLE
Fix/stripe minor fixes

### PR DESCRIPTION
**Pull request by moraleida**
### 🎫 Ticket

[ET-1371] 

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [ ] My code is tested. 
- [ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1371]: https://theeventscalendar.atlassian.net/browse/ET-1371?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by codingmusician — DISMISSED** [review]

File: src/resources/js/commerce/gateway/stripe/checkout.js [hunk 1]
```diff
@@ -345,6 +345,7 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 			obj.cardCvc.mount( obj.selectors.cardCvc );
 			var cardZipWrapper = document.querySelector( obj.selectors.cardZipWrapper );
 			var zipField = document.createElement( 'input' );
+			zipField.placeholder = 'Zip Code';
```
Comment (rafsuntaskin) [inline src/resources/js/commerce/gateway/stripe/checkout.js:348]: We need to add localization support for this placeholder.

File: src/views/tickets/email.php [hunk 2]
```diff
@@ -414,7 +414,8 @@
 											<tr>
 												<td valign="top" align="center" width="100%" style="padding: 0 !important; margin:0 !important;">
 													<h2 style="color:#0a0a0e; margin:0 0 10px 0 !important; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-style:normal; font-weight:700; font-size:28px; letter-spacing:normal; text-align:left;line-height: 100%;">
-														<a style="color:#0a0a0e !important" href="<?php echo esc_url( $event_link ); ?>"><?php echo $event->post_title; ?></a>
+														<a style="color:#0a0a0e !important" href="<?php echo esc_url( $event_link ); ?>">
+															<?php echo $event->post_title; ?></a>
```
Comment (bordoni) [inline src/views/tickets/email.php:418]:
```suggestion
														<a style="color:#0a0a0e !important" href="<?php echo esc_url( $event_link ); ?>">
															<?php echo $event->post_title; ?>
														</a>
```

File: src/resources/js/commerce/gateway/stripe/checkout.js (hunk 1 above)
Comment (bordoni) [reply src/resources/js/commerce/gateway/stripe/checkout.js:348]: @moraleida we should address this before merging.

**Review by bordoni — CHANGES_REQUESTED** [review]
Two small changes and we are good

File: src/resources/js/commerce/gateway/stripe/checkout.js (hunk 1 above)
Comment (moraleida) [reply src/resources/js/commerce/gateway/stripe/checkout.js:348]: @bordoni @rafsuntaskin this is just a placeholder for the code @codingmusician will be adding to deal w/ those fields. I expect this to be completely reworked in the front end

**Review by codingmusician — APPROVED** [review]

**Review by bordoni — APPROVED** [review]

File: src/Tickets/Commerce/Gateways/Stripe/Assets.php [hunk 3]
```diff
@@ -71,6 +71,9 @@ public function register() {
 							'cardElementType' => tribe( Stripe_Elements::class )->card_element_type(),
 							'publishableKey' => tribe( Merchant::class )->get_publishable_key(),
 							'paymentIntentData' => tribe( Client::class )->get_publishable_payment_intent_data(),
+							'i18n' => array(
+								'zip_code' => __( 'Zip Code', 'event-tickets' ),
+							)
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/Stripe/Assets.php:76]: We use short syntax for arrays.

**Pull request merged**